helm/fluent-logshipping-app/values.yaml

```diff
@@ -11,7 +11,7 @@ project:
   commit: "[[ .SHA ]]"
 
 registry:
-  domain: docker.io
+  domain: quay.io
```

Comment:
How is this related? Shouldn't the image be on both quay and docker?

Reply:
It's not, i had to do it during testing because I cannot delete tags on docker